Documentation/ExtensionArchitecture/ExtensionManagement/Index.rst

```diff
@@ -40,9 +40,9 @@ There are only two (possibly three) steps involved in using extensions with TYPO
    The Extension Manager (EM) should take care of this process, including updates to
    newer versions if needed.
 
-   Another convenient way to install extensions is offered by using composer (https://getcomposer.org/).
-   Besides TYPO3 CMS itself the TYPO3 composer repository includes all TYPO3 Extensions that are uploaded to TER.
-   Read more on https://composer.typo3.org/ .
+   Another convenient way to install extensions is offered by using Composer (https://getcomposer.org/)
+   along with TYPO3 Composer Repository (https://composer.typo3.org/). The TYPO3 Composer Repository
+   includes all TYPO3 extensions that are uploaded to TER as well as TYPO3 CMS itself.
```

Comment:
The last part is not true, you cannot find TYPO3 nor its system extensions in TER.

Reply:
The "as well as" refers to the TYPO3 CMS being available in TYPO3 Composer Repository (in addition to the extensions from TER). I guess it's confusing then? Would _"The TYPO3 Composer Repository includes all TYPO3 extensions that are uploaded to TER and the TYPO3 CMS itself."_ be more clear?

Reply:
Not really, since you cannot really upload extensions to TYPO3 CMS. ;-)

The current version is the most precise:

> ... the TYPO3 composer repository includes all TYPO3 Extensions that are uploaded to TER

That is all that composer.typo3.org provides, nothing else.

Reply:
I see :-) I dropped the reference to TYPO3 CMS completely.